schema.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE animals (
+
+    id: integer
```

Comment:
[OPTIONAL]
- [ ] I would advice that you make the *id column not null and autoincremented*, see the below snippet as a guide
`id int NOT NULL IDENTITY(1,1)`
- [ ] You can also mark the id column as primary key,  see the below snippet as a guide
`PRIMARY KEY (id)`